controllers/hardwareclassification_controller.go

```diff
@@ -33,24 +40,39 @@ type HardwareClassificationReconciler struct {
 	Scheme *runtime.Scheme
 }
 
+// Reconcile reconcile function
 // +kubebuilder:rbac:groups=metal3.io,resources=hardwareclassifications,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=metal3.io,resources=hardwareclassifications/status,verbs=get;update;patch
+func (hcReconciler *HardwareClassificationReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
+	ctx := context.Background()
+
+	// Initialize the logger with namespace
+	hcReconciler.Log = hcReconciler.Log.WithName(HWControllerName).WithValues("metal3-harwdwareclassification", req.NamespacedName)
 
-// Add RBAC rules to access baremetalhost resources
-// +kubebuilder:rbac:groups=metal3.io,resources=baremetalhosts,verbs=get;list;update
-// +kubebuilder:rbac:groups=metal3.io,resources=baremetalhosts/status,verbs=get
```

Comment:
I don't think this should be deleted, just moved to line 45 maybe ?

Reply:
Good to address this in PR #25. 
You fork is in sync with upstream master, please update your fork.

Reply:
After rebase changes this is also handled here only.